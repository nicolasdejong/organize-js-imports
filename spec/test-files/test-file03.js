// Test javascript to parse imports

import 'a'
import 'b'

  import c from 'libc'
import d from 'libd'

import 'e'
function abc() {
  let abc = 123;
  let regex = /some|import|regex/g;
}

import 'f'
const def = 456;


import j, { k, l, m } from 'foo'
function ghi() {
  let ghi = 456;
}

import {o} from './../opq'
import {p} from '../opq'
import {q} from '../opq'
import r from '../rst'