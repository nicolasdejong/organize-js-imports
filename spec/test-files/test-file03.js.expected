// Test javascript to parse imports

import 'a';
import 'b';
import 'e';
import 'f';

import j, {k, l, m} from 'foo';
import c            from 'libc';
import d            from 'libd';

import {o, p, q}    from '../opq';
import r            from '../rst';

function abc() {
  let abc = 123;
  let regex = /some|import|regex/g;
}

const def = 456;


function ghi() {
  let ghi = 456;
}

